sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -395,3 +450,16 @@ def test_rfe_allow_nan_inf_in_x(cv):
         rfe = RFE(estimator=clf)
     rfe.fit(X, y)
     rfe.transform(X)
+
+
+def test_w_pipeline_2d_coef_():
```

Comment:
Do you have same for SelectFromModel?

Reply:
Almost similar with slight modifications.

Reply:
I see. didn't notice  that was multiclass